### PR TITLE
Create main.yml

### PR DESCRIPTION
**Pull request by decanus**
possible method to remove external check dependencies by using github actions.

### CODE REVIEW

**Comment by djrtwo** [comment]
cc @mpetrunic 

proposal to replace circleCI with a github action.
I'm kind of in the "if it ain't broke, don't fix it camp", but don't really care too much

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (mpetrunic) [inline .github/workflows/main.yml:12]: I wouldn't restrict it, I would let it run on every push and PR

**Comment by decanus** [comment]
@mpetrunic fixed, @djrtwo ready to merge!

**Comment by djrtwo** [comment]
okay, just disabled circleci build

**Pull request merged**